src/containers/Board/SymbolDetails/SymbolDetails.js

```diff
@@ -138,28 +186,19 @@ export class SymbolDetails extends Component {
 SymbolDetails.propTypes = {
   open: PropTypes.bool,
   onCancel: PropTypes.func,
-  symbol: PropTypes.object
+  editingSymbol: PropTypes.array
 };
 
 SymbolDetails.defaultProps = {
-  symbol: {
-    type: 'symbol'
-  }
+  editingSymbols: []
 };
 
 const mapStateToProps = state => {
-  return {
-    selectedSymbols: state.board.selectedSymbols
-  };
+  return {};
```

Comment:
We don't need anything from the store. selectedSymbols wasn't use anyway.
I don't know if it can stays like that or we should just not connect this component

Reply:
I'll check later if we can remove connect